### PR TITLE
Test Scripts - Refactoring of Common modules

### PR DESCRIPTION
**Pull request by dboltovskyi**
[ATF test scripts](https://github.com/smartdevicelink/sdl_atf_test_scripts/) use a lot of functions in commonly used modules, like:

  - data accessors: file, database
  - functions for tables: print, copy
  - sequences:
    - create mobile session
    - register/activate mobile application
    - perform policy table update
  - etc.

All these functionality are not tests artifacts but tools that tests use.

The purpose of this proposal is to clean up these common modules that everyone is able to use while creating ATF test scripts.

### CODE REVIEW

**Comment by AStasiuk** [comment]
@theresalech proposal is ready for review

**Comment by dboltovskyi** [comment]
@theresalech PR description is updated. Please check.

**Comment by dboltovskyi** [comment]
@theresalech Additional sentence have been added. Please check.

File: proposals/NNNN-test-scripts-common-modules-refactoring.md [hunk 1]
```diff
@@ -19,6 +19,8 @@
 
 All these functionality are not tests artifacts but tools that tests use.
 
+The purpose of this proposal is to clean up these common modules that everyone be able to use them while creating ATF test scripts.
```
Comment (theresalech) [inline proposals/NNNN-test-scripts-common-modules-refactoring.md]: Please update to "...clean up these common modules that everyone is able to use while creating ATF test scripts."

Comment (dboltovskyi) [reply proposals/NNNN-test-scripts-common-modules-refactoring.md]: @theresalech Proposal and PR's description has been updated as you suggested. 
Thank you for the review.

**Pull request merged**